### DISCUSSION TITLE
1.12 to 1.13 breaking changes

### DISCUSSION THREAD

**Discussion by ber4444** [patrykandpatrick/vico#485] [Questions] [answered]
Before:
<img width="340" alt="Screenshot 2023-12-08 at 2 45 50 PM" src="https://github.com/patrykandpatrick/vico/assets/205867/04c4c853-3a29-41bc-a364-ccb64a334707">
After:
<img width="362" alt="Screenshot 2023-12-08 at 2 23 29 PM" src="https://github.com/patrykandpatrick/vico/assets/205867/77f61e24-efa1-43f0-bab1-dd29661a611b">
Because of breaking API changes, we needed to integrate, our attempt was:
<img width="941" alt="Screenshot 2023-12-08 at 2 46 46 PM" src="https://github.com/patrykandpatrick/vico/assets/205867/86aa2123-98cc-41fb-b888-3c99fa871fb3">
<img width="713" alt="Screenshot 2023-12-08 at 2 46 57 PM" src="https://github.com/patrykandpatrick/vico/assets/205867/eb4793bb-068a-4a5c-a169-f93150f4da21">

**Comment by patrickmichalik** [comment, marked as answer]
Hello! It looks like you’ve swapped the data sets. With `multiChartEntryModelProducer + chartEntryModelProducer`, the multi-series data set comes first. In the second code snippet, the multi-series data set is added after the single-series one. Please try switching the two lines, so the order is preserved:

```kt
composedChartEntryModelProducer.runTransaction {
    add(assets, liabilities)
    add(net)
}
```
